plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/BridgeVifDriver.java

```diff
@@ -298,11 +298,11 @@ public void detach(LibvirtVMDef.InterfaceDef iface) {
         Script.runSimpleBashScript("test -d /sys/class/net/" + iface.getBrName() + "/brif/" + iface.getDevName() + " && brctl delif " + iface.getBrName() + " " + iface.getDevName());
     }
 
-    private String setVnetBrName(String pifName, String vnetId) {
+    private String getVnetBrName(String pifName, String vnetId) {
```

Comment:
i dont think getVnetBrName() is a good name, what it does is format/generate/create a vnetBrName! not get it from somewhere.

Reply:
Yes, that is to say, we may also name these methods as createXXX.
Right?